### PR TITLE
Set up physical device, add manual start and move start time later

### LINKED ISSUE #15: Run tests on many versions via FireBase

### LINKED ISSUE #107: Add ability to start tests on Firebase manually
Now we can start the Firebase's pipeline manually only after a previous automatic start which occures in certain time (UTC+0). It is not so convinient and it is possible to wait more than 24 hours to test the pipeline. So a manual start can be helpful.

### LINKED ISSUE #109: Run FireBase on physical devices
Follow up #15 
Chosen device Pixel2 was virtual, should be fixed

### PR DESCRIPTION
**Pull request by greenEkatherine**
Fix #107 
Fix #109

### CODE REVIEW

File: .github/workflows/firebase.yml [hunk 1]
```diff
@@ -1,9 +1,10 @@
 name: FireBase
 
 on:
+  workflow_dispatch:
   schedule:
-    # run every day at 01:00 UTC+0
-    - cron:  '0 1 * * *'
+    # run every day at 04:00 UTC+0
+    - cron:  '0 4 * * *'
```
Comment (omen273) [inline .github/workflows/firebase.yml:7]: Why 4?

File: .github/workflows/firebase.yml [hunk 2]
```diff
@@ -61,4 +62,4 @@ jobs:
           gcloud firebase test android models list
           gcloud firebase test android run --type instrumentation --use-orchestrator \
           --app app-debug/app-debug.apk --test app-debug-androidTest/app-debug-androidTest.apk \
-          --device model=Pixel2,version=28,locale=en,orientation=portrait
\ No newline at end of file
+          --device model=blueline,version=28,locale=en,orientation=portrait
```
Comment (omen273) [inline .github/workflows/firebase.yml:65]: What is blueline? Please, add comment.

File: .github/workflows/firebase.yml (hunk 1 above)
Comment (greenEkatherine) [reply .github/workflows/firebase.yml:7]: Because you wanted less busy time 😃

**Review by omen273 — CHANGES_REQUESTED** [review]
Please address my comments

**Review by omen273 — APPROVED** [review]

**Pull request merged**